recipes/prometheus-cpp/all/test_package/CMakeLists.txt

```diff
@@ -5,8 +5,8 @@ find_package(prometheus-cpp CONFIG REQUIRED)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
 target_link_libraries(${PROJECT_NAME} PRIVATE prometheus-cpp::push prometheus-cpp::pull)
-if(${prometheus-cpp_VERSION} VERSION_LESS "1.1.0")
```

Comment:
You could simplify all these line to `target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_14)` only. C++14 is compatible with C++11, so it should not break the test package, so we don't need this condition.

Reply:
I believe `cxx_std_11` is required in all versions except 1.1.0.
To simplify this code, should I use `target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)` in all versions?